src/Weather/PCMet/Images.cpp

```diff
@@ -118,44 +129,46 @@ PCMet::DownloadLatestImage(const char *type, const char *area,
   Net::DownloadToBufferJob job(session, url, buffer, sizeof(buffer) - 1);
   job.SetBasicAuth(username, password);
   if (!runner.Run(job))
-    return Bitmap();
+    return bitmap;
 
   buffer[job.GetLength()] = 0;
 
   static constexpr char img_needle[] = "<img name=\"bild\" src=\"/";
-  char *img = strstr(buffer, "<img name=\"bild\" src=\"/");
+  char* img = strstr(buffer, "<img name=\"bild\" src=\"/");
   if (img == nullptr)
-    return Bitmap();
+    return bitmap;
 
-  char *src = img + sizeof(img_needle) - 2;
-  char *end = strchr(src + 1, '"');
+  char* src = img + sizeof(img_needle) - 2;
+  char* end = strchr(src + 1, '"');
```

Comment:
More code reformatting

Reply:
I have to avoid the changes by my editor!